src/middle_level/loader/GLTF2Loader.js

```diff
@@ -1,3 +1,7 @@
+import draco3dgltf from 'draco3dgltf';
+console.log(draco3dgltf)
```

Comment:
@emadurandal 
Tell me why draco3dgltf is undifined after 'npm run build' ?
Why rollup does not include draco3dgltf?
Where is the include settings?

Reply:
Because It's CommonJS module, not ES module.